proto/api/v1/api.proto

```diff
@@ -130,6 +130,7 @@ message Processor {
 
   string type = 4 [(google.api.field_behavior) = IMMUTABLE];
   Parent parent = 6 [(google.api.field_behavior) = IMMUTABLE];
+  string condition = 9;
```

Comment:
We should add some docs about the condition (i.e. that it's a Go template and whatever else is useful). I believe that adding the docs here would also put them into the Swagger docs.